schema.sql

```diff
@@ -0,0 +1,43 @@
+CREATE DATABASE comments;
```

Comment:
I would recommend having a 'drop database'  code such as the one I put in below. This way, when we make edits directly on this schema.sql file, we don't have to manually drop the database through our terminal.  

`-- create database comments (drop if exists)
DROP DATABASE IF EXISTS comments;`